applications/serial_lte_modem/src/lwm2m_carrier/slm_at_carrier.c

```diff
@@ -153,18 +153,33 @@ int lwm2m_carrier_event_handler(const lwm2m_carrier_event_t *event)
 	switch (event->type) {
 	case LWM2M_CARRIER_EVENT_LTE_LINK_UP:
 		LOG_DBG("LWM2M_CARRIER_EVENT_LTE_LINK_UP");
+#if defined(SLM_CARRIER_FOTA_AUTOMATION)
```

Comment:
```suggestion
#if defined(CONFIG_SLM_CARRIER_FOTA_AUTOMATION)
```
Same for the other instances of this.